modules/core/src/main/java/org/apache/ignite/internal/commandline/CommandHandler.java

```diff
@@ -222,13 +225,18 @@ public int execute(List<String> rawArgs) {
 
         String commandName = "";
 
+        Throwable err = null;
+        boolean printErrStackTrace = false;
+
         try {
             if (F.isEmpty(rawArgs) || (rawArgs.size() == 1 && CMD_HELP.equalsIgnoreCase(rawArgs.get(0)))) {
                 printHelp();
 
                 return EXIT_CODE_OK;
             }
 
+            printErrStackTrace = rawArgs.stream().anyMatch(CMD_PRINT_ERR_STACK_TRACE::equals);
```

Comment:
May be more proper use F.exist(rawArgs, (arg) -> CMD_PRINT_ERR_STACK_TRACE.equals(CMD_PRINT_ERR_STACK_TRACE))

Reply:
I think to use parser instead of go through console args manually.

Reply:
Remade to F#exist. 
In the beginning I did through the parser, but then I realized that while parsing can also appear error, stack where you want to watch, so I made also the argument "--print-err-stack-trace" may be at the end and it not get.